### ISSUE TITLE
one-level network for 64x64 image size datasets

### ISSUE THREAD

**Issue by jcpeterson** [igul222/PixelVAE#2]
Any chance you can add this?

**Comment by igul222** [comment]
I can put something together, but we ran experiments with this and found it didn't perform nearly as well as 2-level. Do you want it anyway?

**Comment by jcpeterson** [comment]
Yes, that would be much appreciated. I'd like to compare performance on a few datasets under several conditions, especially celebA, which I can already successfully load in as jpg files. Thanks

**Comment by jcpeterson** [comment]
It doesn't have to run with this codebase if you have another that works already

**Comment by igul222** [comment]
Just updated the code with a model that should work ('64px_big_onelevel'), but I haven't actually tested it. Let me know how it goes if you do.

**Comment by jcpeterson** [comment]
Thanks Ishaan! Likely citation incoming ;)

I will give this a shot and let you know if it runs / how it performs.

I don't know if you are aware, but this architecture seems quite SOTA for smallish datasets with manageable variation (more complex than MNIST, but less than CIFAR where data-to-complexity ratio seems off to me), outperforming even the best new GANs. That doesn't really come across in the paper.

**Comment by igul222** [comment]
Interesting! SOTA in what sense, sample quality? I'd say LSUN bedrooms fits the description you mentioned (plenty of data, not terribly complex images) and we tried to demonstrate pretty good results there I think.

________________________________

**Comment by jcpeterson** [comment]
Sample quality, but also a lower probability of artifacts that give away that the images aren't "real". Agreed on LSUN, and clearly you tried a large range of datasets, much more than most papers.

I can show you some examples soon enough.

**Comment by jcpeterson** [comment]
Ok, it looks like training works with the new script, thanks! The sampling code does seem different though. The samples are int32 (isn't this wrong?), and it samples 8 points and then adds variability to those?

**Comment by jcpeterson** [comment]
Actually, the int32 is for the output images it seems, so that's fine. However, I'm still not sure where the variability is coming from in the rows of the output image

**Comment by igul222** [comment]
Can you point to the sampling code you're talking about? The code that gets executed should be lines 855-902.

**Comment by jcpeterson** [comment]
Yes that's right. When this runs, I get 8 samples (rows), and then 8 variations (columns) on those samples. I'd like sampling behavior like the two-level network with the `logits *= 10000.` line added. The redundant columns are easy to remove, so that's not really a problem.

**Comment by jcpeterson** [comment]
Any update on this? Sorry to bother!

**Comment by jcpeterson** [comment]
Any update on this? Sorry to bother!
